specs/wp-signup-spec.js

```diff
@@ -935,11 +937,9 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 					checkOutPage = await CheckOutPage.Expect( driver );
 				} catch ( err ) {
 					//TODO: Check this code once more when domain registration is not available
-					const securePaymentComponent = await SecurePaymentComponent.Expect( driver );
-					let numberOfItems = await securePaymentComponent.numberOfProductsInCart();
-					if ( numberOfItems === 0 ) {
+					if ( driverHelper.isEventuallyPresentAndDisplayed( driver, By.css( '.empty-content' ) ) ) {
 						await SlackNotifier.warn(
-							'OOPS! Something went wrong! Check if domains registrations is available.'
+							'OOPS! Something went wrong, you don\'t have any site! Check if domains registrations is available.'
```

Comment:
Change this to something like 'OOPS! Something went wrong, you don\'t have a site! Check if domains registrations is available.'